response_operations_ui/__init__.py

```diff
@@ -6,12 +6,12 @@
 from flask import Flask, flash, redirect, session, url_for
 from flask_assets import Environment
 from flask_login import LoginManager
-from flask_session import Session
 from flask_talisman import Talisman
 from flask_wtf.csrf import CSRFError, CSRFProtect
 from structlog import wrap_logger
 
 from config import Config
+from flask_session import Session
```

Comment:
ah, this'll fail the linting btw.  The reason it's happening is that I think running the unit tests at the top level creates a `flask_session` folder to store the session locally.  isort sees this and goes 'oh flask_session folder, that must mean it's a local, not a 3rd party package' and puts it with all the other local ones.

Reply:
Bit of an odd one that, and naturally it passes local. Thanks